tests/multi_can/remote_req.py

```diff
@@ -0,0 +1,74 @@
+from machine import CAN
```

Comment:
Maybe these tests can go in `tests/multi_extmod/` and be prefixed with `machine_can_`?  Tests in that directory most likely require hardware connections anyway, and if you just want to run CAN tests then can do:
```
$ ./run-multitests.py -t a0 -t a1 multi_extmod/machine_can_*.py
```